redis_inventory.py

```diff
@@ -25,10 +26,10 @@ def get_product(self, product_id):
 
 
   def get_all(self):
-    result = {}
+    result = []
     for key in self.redis.keys():
       if key != 'index':  # filer out our id index
-        result[key] = self.redis.hgetall(key)
+        result.append(self.redis.hgetall(key))
```

Comment:
I think we also need to add product_id to the info in create_product()

Or there will not be the product_id field.